src/main/resources/application.conf

```diff
@@ -139,6 +139,12 @@ grothendieck {
     # Doc: https://github.com/ethereum/EIPs/issues/150
     eip150-block-number = "2500000"
 
+    # EIP-155 fork block number
+    # Doc: https://github.com/ethereum/eips/issues/155
+    # 3 000 000 because of
+    #https://github.com/paritytech/parity/blob/b50fb71dd1d29dfde2a6c7e1830447cf30896c31/ethcore/res/ethereum/classic.json#L15
```

Comment:
It's ok to say 'Following lead of existing clients implementation to maintain compatibility <link>'. Which I think is what we did here in the absence of a clearly agreed spec....?